pythonforandroid/recipes/python2/__init__.py

```diff
@@ -5,41 +5,101 @@
+               'patches/fix-locale-{version}.patch',
+               'patches/modules-locales-{version}.patch',
+               'patches/fix-platform-{version}.patch',
+               'patches/fix-gethostbyaddr.patch',  # OLD 2.7.2 Patch
+               'patches/basic-android-{version}_mod.patch',
+
+               # APPLY OLD WORKING 2.7.2 PATCHES
                'patches/fix-filesystemdefaultencoding.patch',
                'patches/fix-termios.patch',
                'patches/custom-loader.patch',
-               'patches/verbose-compilation.patch',
                'patches/fix-remove-corefoundation.patch',
                'patches/fix-dynamic-lookup.patch',
                'patches/fix-dlfcn.patch',
-               'patches/parsetuple.patch',
                'patches/ctypes-find-library-updated.patch',
-               ('patches/fix-configure-darwin.patch', is_darwin),
-               ('patches/fix-distutils-darwin.patch', is_darwin),
-               ('patches/fix-ftime-removal.patch', is_api_gt(19)),
-               ('patches/disable-openpty.patch', check_all(is_api_lt(21), is_ndk('crystax')))]
+
+               # Todo: Regular Patches from 2.7.2, Not needed on 2.7.9, must be removed?
+               # 'patches/Python-{version}-ctypes-disable-wchar.patch',
+               # 'patches/disable-modules.patch',
+               # 'patches/fix-setup-flags.patch',
+               # 'patches/verbose-compilation.patch',
+
+               # Todo: Untested Special Patches from 2.7.2, Will work on 2.7.9?
+               # ('patches/fix-configure-darwin.patch', is_darwin),
+               # ('patches/fix-distutils-darwin.patch', is_darwin),
+               # ('patches/fix-ftime-removal.patch', is_api_gt(19)),
+               # ('patches/disable-openpty.patch', check_all(is_api_lt(21), is_ndk('crystax')))
+               ]
 
     from_crystax = False
 
-    def build_arch(self, arch):
+    def prebuild_arch(self, arch):
+        # Here we update some patches to our build system
+        if not exists(join(self.get_build_dir(arch.arch), '.patched')):
+            recipe_dir = self.get_recipe(self.name, arch.arch).recipe_dir
+
+            # Creates a copy of xcompile.patch...to be adapted to our build
+            patch_xcompile_base = join(recipe_dir, 'patches/Python-{0}-xcompile.patch'.format(self.version))
+            patch_xcompile_mod = join(recipe_dir, 'patches/Python-{0}-xcompile_mod.patch'.format(self.version))
+            shutil.copy(patch_xcompile_base, patch_xcompile_mod)
 
+            # Creates a copy of basic-android.patch...to be adapted to our build
+            patch_android_base = join(recipe_dir, 'patches/basic-android-{0}.patch'.format(self.version))
+            patch_android_mod = join(recipe_dir, 'patches/basic-android-{0}_mod.patch'.format(self.version))
+            shutil.copy(patch_android_base, patch_android_mod)
+
+            # REWRITES PATCH TO MATCH SSL DIRS
+            if 'openssl' in self.ctx.recipe_build_order:
```

Comment:
Ok, I took your advice,now I have made a different approach in my local system (now I rewrite directly the affected  files, after the patch has been applied)...I would like to try the changes, before I push changes into master branch.